etlman/static/js/debug.js

```diff
@@ -0,0 +1,11 @@
+htmx.defineExtension('debug', {
```

Comment:
Are these copied from somewhere? Perhaps include a comment stating how to update them in the future, if so.

Reply:
Great idea. Yes, they are.